Fix asserts in tests

H-Huang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#72864 Fix asserts in tests**

Fixes #72860

Differential Revision: [D34246987](https://our.internmc.facebook.com/intern/diff/D34246987)

H-Huang committed:
Fix asserts in tests

[ghstack-poisoned]

code-review-doctor commented:
@H-Huang very interesting timing :)

was it coincidence we both tried fixing this issue within 10 mins of each other?

H-Huang commented:
@H-Huang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34246987).

H-Huang committed:
Update on "Fix asserts in tests"


Fixes #72860

Differential Revision: [D34246987](https://our.internmc.facebook.com/intern/diff/D34246987)

[ghstack-poisoned]

rohan-varma reviewed (approved):
LGTM
